tune_sklearn/_trainable.py

```diff
@@ -86,8 +91,13 @@ def _train(self):
                     self.y,
                     test,
                     train_indices=train)
-                self.estimator[i].partial_fit(X_train, y_train,
-                                              np.unique(self.y))
+                if hasattr(self.estimator, "partial_fit"):
+                    self.estimator[i].partial_fit(X_train, y_train,
+                                                  np.unique(self.y))
+                else:
+                    simplefilter("ignore", category=ConvergenceWarning)
```

Comment:
this seems a little scary; maybe output it once but catch the rest of them?

Reply:
Actually I wasn't able to get this working yet I think all the errors still show.